neurobooth_os/tasks/task.py

```diff
@@ -366,6 +368,103 @@ def gaze_contingency():
         pass
 
 
+class Color(Enum):
+    BLACK = 0
+    BLUE = 1
+    GREEN = 2
+    CYAN = 3
+    RED = 4
+    MAGENTA = 5
+    BROWN = 6
+    LIGHTGRAY = 7
```

Comment:
Not necessary, but I think adding underscores would help readability. E.g., LIGHT_GRAY

Reply:
Sticking to the same format as source documentation